src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -1,8 +1,51 @@
 package frc.robot.subsystems;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.Ports;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
 
 public class Intake extends SubsystemBase {
-    // TODO: Create empty intake methods (e.g. move up, move down, run rollers, etc.)
-    // TODO: start implementing methods
-}
\ No newline at end of file
+
+    private final CANSparkMax m_rollerMotor = new CANSparkMax(Ports.ROLLER_MOTOR_PORT, MotorType.kBrushed); // may be                                                                                                        // not sure                                                                                                          // rn
+    private final CANSparkMax m_intakeMotor = new CANSparkMax(Ports.INTAKE_MECHANISM_MOTOR_PORT, MotorType.kBrushed); // ^^
+    private double intakeMotorSpeed = 0.5; 
+    private double rollerMotorSpeed = 0.5;
+    public static IntakePosition currentIntakePosition = IntakePosition.GROUND;
```

Comment:
First, very rarely are static methods and fields the right choice. You should only use static methods for functions that will never change, like calculating the sine of angle or similar, and even then sometimes non-static methods are better. Here you definitely don't want this to be static. The better way to do what you were trying to do is to make this a regular field, and pass the instance of the `Intake` class in through the constructor of `MoveIntake`.

The reasons for not using static methods even though there's only one intake require a lot more explanation and experience to fully understand, but basically the reason is it allows you to test `MoveIntake` independently of `Intake`, and makes it easier to make changes to the program.

But also, I wouldn't store the `currentIntakePosition` at all. The point of a subsystem is to interface with the hardware of the robot. Right now, `currentIntakePosition` doesn't actually store the current intake position, it actually stores where you last told the intake to go. But that's not a hardware concern, that's a *user interface* concern. So, the subsystem probably shouldn't store this information. And, there's no reason it needs to know this information. It can make decisions based on the actual measured robot state, not what we told it to.